preprocessing/nextclade/tests/factory_methods.py

```diff
@@ -36,6 +36,7 @@ def create_unprocessed_entry(
             accessionVersion=f"LOC_{accession_id}.1",
             data=UnprocessedData(
                 submitter="test_submitter",
+                group_id=1,
```

Comment:
it might be good to use group_id=2 as 1 will always be the insdc_ingest group

Reply:
done!